local/owr_media_renderer.c

```diff
@@ -308,6 +308,25 @@ static void on_caps(GstElement *sink, GParamSpec *pspec, OwrMediaRenderer *media
     }
 }
 
+void owr_media_renderer_set_state(OwrMediaRenderer *_renderer,MediaRenderState state)
+{
+    OwrMediaRenderer* renderer;
+    OwrMediaRendererPrivate *priv;
+
+    renderer = OWR_MEDIA_RENDERER(_renderer);
+    priv = renderer->priv;
+
+    switch(state){
```

Comment:
[Style] This switch statement does not follow the syntax of the rest of the code. Please have a look at other similar statements.